client/me/security-checkup/recovery-phone.jsx

```diff
@@ -1,51 +1,53 @@
 /**
  * External dependencies
  */
-var React = require( 'react' ),
-	assign = require( 'lodash/assign' ),
-	isEmpty = require( 'lodash/isEmpty' );
+
+import React, { Component } from 'react';
+import isEmpty from 'lodash/isEmpty';
```

Comment:
We can now just do `import { isEmpty } from 'lodash'`. I believe tree shaking is not far off